commands/login.go

```diff
@@ -67,4 +62,7 @@ func (cmd *Login) Run(scope scope.Scope, c *cli.Context) {
 
 	cmd.config.Map["target"] = cmd.network.BrooklynUrl
 	cmd.config.Write()
+	
```

Comment:
I tried only writing the config if the login succeeds, but this has a bad side affect; if the login fails, the previous credentials are still enabled in the config file and potentially pointing to the wrong brooklyn node.  Need more thought on what would be the best approach.